src/Microsoft.IdentityModel.Tokens/AsymmetricSignatureProvider.cs

```diff
@@ -134,15 +126,15 @@ public AsymmetricSignatureProvider(SecurityKey key, string algorithm, bool willC
         }
 
         /// <summary>
-        /// Gets the mapping from algorithm to the minimum <see cref="AsymmetricSecurityKey"/>.KeySize for creating signatures.
+        /// Gets the mapping from algorithm to the minimum <see cref="SecurityKey.KeySize"/> for creating signatures.
```

Comment:
We specifically used AsymmetricSecurityKey here as this is the AsymmetricSignatureProvider.

Reply:
Same